python/stack_of_tasks/ui/model/av_ref.py

```diff
@@ -11,15 +12,20 @@
 
 from . import RawDataRole
 
+icon_path = Path(rospkg.RosPack().get_path("rviz")) / "icons/classes"
```

Comment:
Don't pollute the namespace. Keep `icon_path` local to `AvailableRefModel.__init__()`.
Is the path slash (`/`) in `"icons/classes"` correctly resolved (i.e. on Windows using a different separator)?